### ISSUE TITLE
Enabling the "space weather" notification option in difficulty settings does not work

### ISSUE THREAD

**Issue by JonnyOThan** [Kerbalism/Kerbalism#812]
If you turn on the "space weather" notification option in the difficulty settings, it does not affect the initial state of the per-vessel notifications.

KSP Version: 1.12.3
Kerbalism Version (from KerbalismConfig): 3.14.0.0

**Comment by JonnyOThan** [comment]
https://github.com/Kerbalism/Kerbalism/blob/6fc4503048dd766d6d29aece9f167214bdfae24f/src/Kerbalism/Database/VesselData.cs#L685

Does Lib.CrewCount(pv) return 0 maybe?

**Comment by gotmachine** [comment]
Yeah I guess `Lib.CrewCount(pv)` is called to early when the vessel is instantiated for launch, while the stock crew data isn't yet populated.
